Library/Formula/jsvc.rb

```diff
@@ -11,15 +11,15 @@ class Jsvc < Formula
   depends_on :java
 
   def install
-    ENV.append "CFLAGS", "-arch #{MacOS.preferred_arch}"
-    ENV.append "LDFLAGS", "-arch #{MacOS.preferred_arch}"
-    ENV.append "CPPFLAGS", "-I/System/Library/Frameworks/JavaVM.framework/Versions/Current/Headers"
+    ENV.append "CFLAGS", "-arch #{MacOS.preferred_arch}" if OS.mac?
+    ENV.append "LDFLAGS", "-arch #{MacOS.preferred_arch}" if OS.mac?
+    ENV.append "CPPFLAGS", "-I/System/Library/Frameworks/JavaVM.framework/Versions/Current/Headers" if OS.mac?
```

Comment:
In this I would tend to use an `if` block rather than suffix style `if`.
```ruby
if OS.mac?
    ENV.append "CFLAGS", "-arch #{MacOS.preferred_arch}"
    ENV.append "LDFLAGS", "-arch #{MacOS.preferred_arch}"
    ENV.append "CPPFLAGS", "-I/System/Library/Frameworks/JavaVM.framework/Versions/Current/Headers"
end
```